seqr/views/apis/auth_api.py

```diff
@@ -51,6 +51,7 @@ def login_view(request):
 
 def logout_view(request):
     user = request.user
+    remove_token(user)
```

Comment:
I add the removing of the token while logging out because I want the user to be `real` logout from Google authentication. So that when the user logs in locally, he won't be considered as logging in as Google.
It still has a bug here for the local user. I'm fixing it.